app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -158,5 +158,30 @@ public function processInvitationcall($observer)
             }
         }
     }
+    public function sendSyncUrl(): void {
+        $helper = Mage::helper('webwinkelconnect');
+        if (!$helper->isProductReviewInviteEnabled()) {
+            return;
+        }
+
+        $curl = new Varien_Http_Adapter_Curl();
+        $curl->setConfig([
+            'timeout'   => 10,
+        ]);
+        $url = 'https://dashboard.webwinkelkeur.nl/webshops/sync_url';
+        $data = json_encode([
+            'webshop_id' => $helper->getShopId(),
+            'api_key' => $helper->getApiKey(),
+            'url' => Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_WEB) . 'webwinkelkeur/index/sync',
+        ]);
+
+        $curl->write(Zend_Http_Client::POST, $url, '2', ['Content-Type:application/json'], $data);
+        $response = $curl->read();
+        if (!$response) {
```

Comment:
It looks like the HTTP response code isn't checked anywhere.

It's important to test your error handling. In this case you can do that by intentionally commenting out some required POST field or changing the URL path so that you get a 404.

Check whether the error is handled as you intended.

If it turns out that the response code isn't checked (which is what I expect), you can look into enabling [CURLOPT_FAILONERROR](https://www.php.net/curl_setopt).